contrib/completion/zsh/_docker

```diff
@@ -78,7 +78,7 @@ __docker_get_containers() {
             s="${${line[${begin[CONTAINER ID]},${end[CONTAINER ID]}]%% ##}[0,12]}"
             s="$s:${(l:15:: :::)${${line[${begin[CREATED]},${end[CREATED]}]/ ago/}%% ##}}"
             s="$s, ${${${line[${begin[IMAGE]},${end[IMAGE]}]}/:/\\:}%% ##}"
-            if [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Exit* ]]; then
+            if [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Exit* ]] || [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Created* ]] ; then
```

Comment:
Although they do exactly the same thing, your code is a cleaner than mine, so I agree it's probably best to take your version.
Should I create a new pull request with your code or are you guys going to handle the rest ?